sql/src/main/java/io/crate/analyze/TableElementsAnalyzer.java

```diff
@@ -284,6 +285,22 @@ public Void visitIndexDefinition(IndexDefinition node, ColumnDefinitionContext c
             return null;
         }
 
+        @Override
+        public Void visitColumnStorageDefinition(ColumnStorageDefinition node, ColumnDefinitionContext context) {
+            Settings storageSettings = GenericPropertiesConverter.genericPropertiesToSettings(node.properties(),
+                context.parameters);
+
+            for (String property : storageSettings.names()) {
+                if (property.equals("columnstore")) {
```

Comment:
Could we check against a static String variable here (instead of using the hardcoded string)?

Reply:
what reason for?

Reply:
I like constants just to be defined in one place. If you want to change it, you will just have to change it in one place instead of greping thorugh the source to find duplicates.